src/components/calcite-date-picker-day/calcite-date-picker-day.scss

```diff
@@ -99,21 +102,23 @@
 }
 
 :host([selected]) .day {
+  @apply font-medium;
   background-color: var(--calcite-ui-brand) !important;
```

Comment:
`bg-brand`?

Reply:
didn't know if I should convert to tailwind if it was using !important. 

I know how much you like to use `!important`, Paul. :D